simplq/src/components/common/NotificationButton.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import { useSelector, useDispatch } from 'react-redux';
+import { setNotificationPermission } from '../../store/appSlice';
+
+const NotificationButton = (props) => {
+  const dispatch = useDispatch();
+  const notificationPermission = useSelector((state) => state.appReducer.notificationPermission);
+
+  const checkNotificationPromise = () => {
+    try {
+      Notification.requestPermission().then();
+    } catch (e) {
+      return false;
+    }
+
+    return true;
+  };
+
+  const askNotificationPermission = () => {
+    // function to actually ask the permissions
+    const handlePermission = (permission) => {
+      // Whatever the user answers, we make sure Chrome stores the information
+      if (!('permission' in Notification)) {
+        Notification.permission = permission;
+      }
+
+      dispatch(setNotificationPermission(permission));
+    };
+
+    // Let's check if the browser supports notifications
+    if (!('Notification' in window)) {
+      console.log('This browser does not support notifications.');
```

Comment:
We could consider doing something more than a console log in this case?

Reply:
We could show this as an error popup. Users won't see the console logs after all

Reply:
we can log for now, and come back to this. popup is one option, or we can put some kind of banner or something that this browser/device doesn't support notifications, or w could just stay silent... let's plan it out

Reply:
create an isssue

Reply:
will do.

Reply:
done